src/main/kotlin/com/terraformation/backend/accelerator/api/GlobalRolesController.kt

```diff
@@ -0,0 +1,75 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.customer.db.UserStore
+import com.terraformation.backend.customer.model.IndividualUser
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.default_schema.GlobalRole
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/globalRoles")
```

Comment:
Slight nit with the placement here: this isn't really accelerator-specific functionality so IMO it belongs outside of `/api/v1/accelerator`. Maybe under `/api/v1/users` instead? Or maybe give it its own top-level name `/api/v1/globalRoles`? And the class can go in the `com.terraformation.backend.customer.api` package.

To Karthik's point about case sensitivity, just as an FYI, you can include multiple values in this annotation, e.g., `@RequestMapping("/api/v1/globalRoles", "/api/v1/globalroles")`. Not saying it's necessary in this case, just pointing out the feature for future reference.

Reply:
Since it is users with global roles, it seems like perhaps it should be `/api/v1/globalRoles/users` for the list and `/api/v1/users/{userId}/globalRoles`. Is it OK for them to have two different top level URL segments in the same file? I could also go with `/api/v1/users/globalRoles` for the list but, to me, semantically it means "the global roles that belong to users".

Reply:
Having two separate top-level paths is fine, I think.